hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/AbstractRecordStore.java

```diff
@@ -49,18 +50,19 @@
  */
 abstract class AbstractRecordStore implements RecordStore<Record> {
 
+    protected final int partitionId;
```

Comment:
I think originally David wanted to keep the same order here as it is initialised in the constructor. Seems reasonable.